zio-cache/shared/src/test/scala/zio/cache/CacheSpec.scala

```diff
@@ -33,6 +33,16 @@ object CacheSpec extends DefaultRunnableSpec {
         } yield assert(contains)(isFalse)
       }
     },
+    testM("invalidateAll") {
+      checkM(Gen.anyInt) { salt =>
+        for {
+          cache <- Cache.make(100, Duration.Infinity, Lookup(hash(salt)))
+          _     <- ZIO.foreach(1 to 100)(cache.get)
```

Comment:
Nitpicking: maybe capture the count here just to show, after `invalidateAll`, the count has changed to 0.

```scala
for {
   cache <- Cache.make(100, Duration.Infinity, Lookup(hash(salt)))
   _     <- ZIO.foreach(1 to 100)(cache.get)
   size1 <- cache.size
   _     <- cache.invalidateAll
   size2 <- cache.size
} yield assertTrue(size1 == 100) && assertTrue(size2 == 0)
```

Reply:
I think we've got other tests, particularly the one for `size` below`, to verify that when we add items to the cache they really are included in the cache and reflected in the cache size so we don't need to test that again here.